cilium-cli/cli/install.go

```diff
@@ -195,6 +195,8 @@ cilium upgrade --set cluster.id=1 --set cluster.name=cluster1
 
 	addCommonInstallFlags(cmd, &params)
 	addCommonHelmFlags(cmd, &params)
+	cmd.Flags().BoolVar(&params.HelmResetThenReuseValues, "reset-then-reuse-values", false,
```

Comment:
This should be enable by default it we want it to be the default behavior.

Reply:
Made changes as requested.